cmd/devdata/main.go

```diff
@@ -340,25 +347,29 @@ func makePlanGeneralCharacteristics(modelPlanID uuid.UUID, logger *zap.Logger, s
 	dbGeneralCharacteristics, _ := store.PlanGeneralCharacteristicsCreate(logger, &gc)
 	return dbGeneralCharacteristics
 
+}
+func makePlanBeneficiaries(modelPlanID uuid.UUID, logger *zap.Logger, store *storage.Store, callbacks ...func(*models.PlanBeneficiaries)) *models.PlanBeneficiaries {
+	b := models.PlanBeneficiaries{
+		ModelPlanID: modelPlanID,
+		CreatedBy:   "ABCD",
+		ModifiedBy:  models.StringPointer("ABCD"),
+		Status:      models.TaskReady,
+	}
+
+	for _, cb := range callbacks {
+		cb(&b)
+	}
+
```

Comment:
And get rid of the `Status` field when making the model above

Though, this will probably get changed soon when we refactor the devdata package, but still probably a good idea!